cms/djangoapps/contentstore/models.py

```diff
@@ -0,0 +1,32 @@
+"""
+Models for contentstore
+"""
+
+from django.db.models.fields import TextField
+
+from config_models.models import ConfigurationModel
+
+
+class VideoEncodingDownloadConfig(ConfigurationModel):
```

Comment:
Creating a new Django model hasn't been the standard way of configuring the UI in Studio. I have seen this done so using Django configuration settings instead. @andy-armstrong or @cahrens Is there a design pattern for configuring UI options in Studio? In this particular case, we want to limit the types (i.e., profiles) and statuses of video encodings that are included in a CSV download UI feature.